### PR TITLE
Import ABC from collections.abc for Python 3.10 compatibility.

### LINKED ISSUE #4011: Resolve DeprecationWarnings Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3
There are few instances of `DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working`:

```
ocs_ci/framework/__init__.py:107
  /home/ocsqe/projects/ocs-ci-unittest/ocs_ci/framework/__init__.py:107: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
    if isinstance(orig, collections.Mapping):

venv/lib64/python3.9/site-packages/jinja2/utils.py:485
  /home/ocsqe/projects/ocs-ci-unittest/venv/lib64/python3.9/site-packages/jinja2/utils.py:485: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
    from collections import MutableMapping

-- Docs: https://docs.pytest.org/en/latest/warnings.html

```

### PR DESCRIPTION
**Pull request by tirkarthi**
Fixes #4011

### CODE REVIEW

**Review by mbukatov — APPROVED** [review]

File: ocs_ci/framework/__init__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 """
 # Use the new python 3.7 dataclass decorator, which provides an object similar
 # to a namedtuple, but allows type enforcement and defining methods.
-import collections
+import collections.abc
```
Comment (clacroix12) [inline ocs_ci/framework/__init__.py:11]: Should we use `from` here?

```suggestion
from collections import abc
```

Comment (tirkarthi) [reply ocs_ci/framework/__init__.py:11]: I just did `from collections.abc import Mapping` since only one ABC is used

**Review by clacroix12 — APPROVED** [review]

**Review by mbukatov — DISMISSED** [review]

**Review by jimdevops19 — DISMISSED** [review]

**Comment by mbukatov** [comment]
Damn github, I didn't intend to merge master into this branch, but wanted to merge it with the conflict resolution into master. Sigh.

**Comment by mbukatov** [comment]
fixed

**Review by mbukatov — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**